src/templates/pycontw-2020/contents/en/speaking/cfp.html

```diff
@@ -18,18 +18,18 @@ <h1>Call for Proposals </h1>
 <p>The Call for Proposals is now open. PyCon 2020 Taiwan is accepting talks and tutorials!</p>
 
 
-<h2>Important Dates(<a href="https://en.wikipedia.org/wiki/Anywhere_on_Earth">We use AoE timezone.</a>)</h2>
+<h2>Important Dates</h2>
 
 <ul>
 	<li>Talks &amp; Tutorial CfP begins: 1 Mar </li>
-	<li>Talks &amp; Tutorial CfP ends: 26 Apr（23:59）</li>
+	<li>Talks &amp; Tutorial CfP ends: 26 Apr 23:59:59 <a href="https://www.timeanddate.com/worldclock/converter.html?iso=20200427T115900&p1=tz_aoe&p2=33&p3=64&p4=136&p5=44&p6=224&p7=235">AoE</a></li>
```

Comment:
I’d suggest to simply put

https://www.timeanddate.com/worldclock/converter.html?iso=20200427T115900&p1=tz_aoe&p2=241&p3=1440

The user can add cities if they wish to.

Reply:
Sure~